third-party/realdds/include/realdds/topics/image/image-msg.h

```diff
@@ -72,6 +72,7 @@ class image
                            eprosima::fastdds::dds::SampleInfo * optional_info = nullptr );
 
     std::vector<uint8_t> raw_data;
+    std::string frame_id;
```

Comment:
We should sync by a `sync_id`, as discussed, using the frame timestamp

Reply:
@maloel will be done in the future or as part of ROS compatibility.
Do you prefer having the current UT implemented with frame_id or as part of the new `sync_id` in the future?